data/events/scripts/monster.lua

```diff
@@ -42,7 +42,21 @@ function Monster:onDropLoot(corpse)
 			end
 		end
 
+		-- Vip system
+		local vipPercentLoot = 0
+		if (configManager.getBoolean(configKeys.VIP_SYSTEM_ENABLED)
+			and configManager.getBoolean(configKeys.VIP_SYSTEM_LOOT_ENABLED)
+			and player and player:isVip()) then
+			vipPercentLoot = (configManager.getNumber(configKeys.VIP_SYSTEM_LOOT_PERCENT) / 100)
```

Comment:
Se VIP_SYSTEM_LOOT_PERCENT for 0, nao vai quebrar o codigo? 0 nao e divisivel, nao e valido criar uma condicao?